pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1844,7 +1849,50 @@ BEGIN
 END
 $func$
 LANGUAGE PLPGSQL;
-GRANT EXECUTE ON PROCEDURE SCHEMA_CATALOG.drop_metric_chunks(text, timestamptz, timestamptz, boolean) TO prom_maintenance;
+GRANT EXECUTE ON PROCEDURE SCHEMA_CATALOG.drop_metric_chunks(text, text, timestamptz, timestamptz, boolean) TO prom_maintenance;
+
+
+-- Get metric hypertable information for raw and cagg metrics
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_metric_hypertable_info(metric_schema_name text, metric_table_name text, is_view boolean)
```

Comment:
nit: makes it more clear it's where the data is stored not the raw hypertable
```suggestion
CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_storage_hypertable_info(metric_schema_name text, metric_table_name text, is_view boolean)
```